src/windows.rs

```diff
@@ -0,0 +1,306 @@
+extern crate libc;
+extern crate kernel32;
+extern crate winapi;
+
+use winapi::{HANDLE, INVALID_HANDLE_VALUE, fileapi, winbase, winnt};
+use winapi::minwinbase::{OVERLAPPED, LPOVERLAPPED};
+use winapi::winerror::ERROR_OPERATION_ABORTED;
+use winapi::winnt::FILE_NOTIFY_INFORMATION;
```

Comment:
All the definitions in the submodules of winapi are pub glob imported into the root module, so there's no need to refer to the submodules. It's only really there for code organization and documentation purposes.